src/main/kotlin/no/nav/helse/spinder/SpinderMatcher.kt

```diff
@@ -0,0 +1,76 @@
+package no.nav.helse.spinder
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import io.prometheus.client.Counter
+import no.nav.helse.Environment
+import no.nav.helse.oppslag.InfotrygdBeregningsgrunnlagOppslag
+import no.nav.helse.oppslag.StsRestClient
+import no.nav.helse.spinder.db.SpinderDb
+import no.nav.helse.streams.defaultObjectMapper
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+
+class SpinderMatcher(val db: SpinderDb, val env: Environment) : Runnable {
+
+    private val stsClient = StsRestClient(baseUrl = env.stsRestUrl, username = env.username, password = env.password)
+
+    private val infotrygd = InfotrygdBeregningsgrunnlagOppslag(env.sparkelBaseUrl, stsClient)
+
+    private val log = LoggerFactory.getLogger(SpinderMatcher::class.java.name)
+
+    private val MATCH = "match";
+    private val MISMATCH = "mismatch";
+    private val INGENDATA = "ingendata";
+    private val SPA_VEDTAK_DESERIALISERINGSFEIL = "spa_vedtak_deserialiseringsfeil";
+    private val INFOTRYGD_OPPSLAGSFEIL = "infotrygd_oppslagsfeil";
+
+    private val matcheCounter = Counter.build()
+        .name("spinder_match_attempts_totals")
+        .labelNames("resultat")
+        .help("antall matcheforsøk gjort, fordelt på $MATCH, $MISMATCH, $INGENDATA (i.e: fant ikke vedtak i infotrygd), $SPA_VEDTAK_DESERIALISERINGSFEIL, $INFOTRYGD_OPPSLAGSFEIL")
+        .register()
+
+    private val minimumWaitMS = 3000
+
+    override fun run() {
+        log.info("Starting SpinderMatcher")
+        while (true) {
```

Comment:
`while (true)` uten noen måte å hoppe ut får meg til å tenke at vi burde løst dette litt annerledes. 
for eksempel å droppe en while (true), og heller hatt en executor-service som jevnlig kjører tråden på nytt